stubs/setuptools/setuptools/__init__.pyi

```diff
@@ -101,7 +101,8 @@ def setup(
     include_dirs: list[str] = ...,
     password: str = ...,
     fullname: str = ...,
-    **attrs,
+    # Custom Distributions could accept more params
+    **attrs: Any,
```

Comment:
Just came across this commit while investigating why `pyright` started warning that `Type of "setup" is partially unknown` since release of `1.1.375` a few weeks ago. Hopefully, this slight addition should fix it.

Reply:
Sounds like it would, yeah.